code/modules/admin/sql_message_system.dm

```diff
@@ -354,12 +406,18 @@
 						alpha = 10
 						skipped = TRUE
 					alphatext = "filter: alpha(opacity=[alpha]); opacity: [alpha/100];"
-
-			var/list/data = list("<p style='margin:0px;[alphatext]'> <b>[timestamp] | [server] | [admin_key]")
+			var/list/data = list("<div style='margin:0px;[alphatext]'>")
+			if(severity)
+				data += "<p class='severity [severity]'>"
+			else
+				data += "<p class='severity'>"
+			data += "<b>[timestamp] | [server] | [admin_key] | [secret ? "<i>- Secret</i>" : ""]"
```

Comment:
You should add a field to display the severity since the color isn't entirely self-evident.